meshio/medit/_medit.py

```diff
@@ -97,11 +101,17 @@ def read_buffer(f):
             if items[0] != "End":
                 raise ReadError("Unknown keyword '{}'.".format(items[0]))
 
-    return Mesh(points, cells, point_data=point_data, cell_data=cell_data)
+    return Mesh(
+        points, cells, point_data=point_data, cell_data=cell_data, info=spec_info,
+    )
 
 
-def write(filename, mesh, float_fmt=".15e"):
+def write(filename, mesh, float_fmt=".15e", info=None):
```

Comment:
No need for an extra info argument. Let's just take `mesh.info`.

Reply:
If I do this
 ```
 m = meshio.read('in.xdmf')
 m.write('out.vtk')
```
I want want the information from `in.xdmf` not be copied into `out.vtk` because it is mostly related to the format. In this case, `written by meshio v4.0.3` should be in the VTK file. The information should only be written if the user explicitly states it as an argument. Do you agree?